### ISSUE TITLE
sass:math, needed for math.div support, not included in bulma-scss 0.9.3?

### ISSUE THREAD

**Issue by pgnd** [j1mc/bulma-scss#20]
After upgrading to
```
bulma-scss '0.9.3'
```

adding to my page scss,
```
page.scss
	@import 'myvars';
+	$test_var: math.div($some-val * 2, 100);
```

then run my usual webpack asset build
```
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: There is no module with the namespace "math".
   ╷
17 │ $test_var: math.div($some-val * 2, 100);
   │            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
```

adding a `use` to the `include`,
```
_myvars.scss
+	@use 'sass:math';
	...
```
doesn't do the trick.  same error as above with webpack build.

but adding directly to
```
page.scss
+	@use 'sass:math';
	@import 'myvars';
+	$test_var: math.div($some-val * 2, 100);
```

webpack completes with no errors.

(1) shouldn't 
```
@use 'sass:math'
```

be 'in' & made available by, bulma sources?

(2) since the `use` doesn't apparently get picked up from the `include`,
is there another method to add in _once_, and not requiring it in each source file?

Since here

https://github.com/jgthms/bulma/issues/3333

it apears to be resolved, it this^  bulma-scss specific?

**Comment by pgnd** [comment]
tl;dr: no.

https://css-tricks.com/introducing-sass-modules/#import-built-in-sass-modules

"There are several built-in modules — math, color, string, list, map, selector, and meta — which have to be imported explicitly in a file before they are used:"
